src/lib/input/input-container.ts

```diff
@@ -71,6 +78,14 @@ export class MdHint {
   @Input() id: string = `md-input-hint-${nextUniqueId++}`;
 }
 
+/** Directive, used to display a single error message under the input. */
+@Directive({
+  selector: 'md-error, mat-error',
```

Comment:
Add these to `compatibility.ts`? It would be nice if we had a check for this, I bet we could use the metadata.json to get all of the selectors already... (cc @DevVersion)